worlds/Arena.wbt

```diff
@@ -66,77 +66,90 @@ Robot {
       territoryName "PNTerritory"
       emitterName "PNEmitter"
       receiverName "PNReceiver"
+      textureName "../textures/territory-labels/PN.png"
```

Comment:
I presume textures can't be SVGs?

Reply:
The Webots docs only mention it supporting PNG and JPG